src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/NextReviewDateUpdaterService.kt

```diff
@@ -93,24 +91,22 @@ class NextReviewDateUpdaterService(
     return nextReviewDatesAfterUpdate
   }
 
-  private suspend fun publishDomainEvents(
+  private fun publishDomainEvents(
```

Comment:
`publishDomainEvents` was called from inside a suspend function and used `runBlocking` – this is not recommended. Since there are no suspend function calls inside `publishDomainEvents`, special coroutine handling can be removed as it likely added unnecessary overhead anyway.